lib/fedex/request/shipment.rb

```diff
@@ -36,21 +36,38 @@ def process_request
 
       # Add information for shipments
       def add_requested_shipment(xml)
+        ship_timestamp = @shipping_options[:ship_timestamp] ||= Time.now.utc.iso8601(2) # Adds the ability to set ship date, for future packages.
```

Comment:
this comment sounds like it's for the reviewers of the PR and not a person reading the comment